databroker/headersource/shim.py

```diff
@@ -18,15 +18,15 @@ def __init__(self, mds):
 
     def __call__(self, text_search=None, filters=None, **kwargs):
         if filters is None:
-            filters = []
+            filters = {}
         else:
-            [_format_time(v, self.mds.config['timezone']) for v in filters]
+            _format_time(filters, self.mds.config['timezone'])
         if text_search is not None:
-            query = {'$and': [{'$text': {'$search': text_search}}] + filters}
+            query = {'$and': [{'$text': {'$search': text_search}}] + [filters]}
```

Comment:
This works, but it's a bit more obfuscated than it needs to be since `filters` is a now dict. This is like saying `[1] + [2]` instead of `[1, 2]`. Ditto below with `query = ...`